test/db/tools/r2

```diff
@@ -125,3 +125,67 @@ EXPECT_ERR=<<EOF
 ERROR: Cannot open empty rapatch path
 EOF
 RUN
+
+NAME=radare2 aaa
+FILE=malloc://123
+CMDS=<<EOF
+-A
+?ee --
+aaa
+?ee --
+r2 -qA malloc://123
```

Comment:
```suggestion
r2 -e scr.color=0 -qA malloc://123
```

try this hopefully that will disable the colors

Reply:
Fixed but keeps displaying weird stuff with the msg `INFO: Analyze function calls (aac)`